### DISCUSSION TITLE
TEMP FLEX - MCDU/PERF/TAKEOFF

### DISCUSSION THREAD

**Discussion by kampinod** [flybywiresim/aircraft#6045] [Q&A] [answered]
Hello, when I change the TEMP TO FLEX, in MCDU / PERF / TAKEOFF, the V1 VR V2 does not vary is this normal? I thought the more we increase the TEMP the more the VR should increase, but I'm not an expert ... PS I'm using the dev version of FBW,

**Comment by andrejetcat** [comment, marked as answer]
Vr do not change with FLEX temperatures. Vr is the speed, where you rotate the plane. Its the same speed for any FLEX setting IRL.
Only V1, the decision speed change a bit IRL, but not Vr.

**Reply by kampinod** [reply]
Thank you very much to you for this information and have a nice day
